fe/fe-core/src/main/java/com/starrocks/load/loadv2/BulkLoadJob.java

```diff
@@ -285,14 +285,14 @@ public void analyze() {
     }
 
     @Override
-    protected void replayTxnAttachment(TransactionState txnState) {
+    protected void replayTxnAttachment(TransactionState txnState, boolean replay) {
```

Comment:
suggest to refactor, this function is misleading since function name is replayTxnAttachment but has parameter control not to replay some thing.